scripts/api-generate.pl

```diff
@@ -39,15 +47,24 @@
         my $html = HTML::TreeBuilder->new(no_space_compacting => 1);
         $html->parse($data);
         $html->eof;
-        my ($section) = map $_->look_down(_tag => 'table'), map $_->right, $html->look_down(_tag => 'h4', sub { shift->as_text =~ /Fields/ });
-
         # Depluralised version of the type - nasty custom logic in here, but given that this isn't
         # ever likely to change we can get away with it for now.
         my $entity_name = ucfirst $type;
         $entity_name = 'Company' if $type eq 'companies';
+        $entity_name = 'Reply' if $type eq 'replies';
         $entity_name =~ s{s$}{};
         $entity_name =~ s{-(.)}{\U$1}g;
 
+        my $section_id = $map_types{$type} ? $type . '-fields' : 'fields';
+        warn $html->look_down(
```

Comment:
Missed this - will remove this in the next branch.

There have been some minor changes to the page layout and I'm not sure that the original classes will regenerate cleanly.